### ISSUE TITLE
[Keros][TagPicker] aria-valid-attr-value failing

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#5044]
#### Describe the issue:
Keros for web fails aria-valid-attr-value check for TagPicker

#### Actual behavior:

Error shown:
Invalid ARIA attribute value: aria-controls="selected-suggestion-alert"

In BasePicker.tsx, aria-controls is currently being set as 'selected-suggestion-alert'.

Result of running Keros for Web:
![image](https://user-images.githubusercontent.com/6390909/40778506-03c0b05c-64ef-11e8-9cc5-2bb81e18a34f.png)
